storage/httpstore.go

```diff
@@ -39,7 +39,14 @@ type NetworkError struct {
 }
 
 func (n NetworkError) Error() string {
-	return n.Wrapped.Error()
```

Comment:
Thank you @cyli ! You are right about this ! I debug it a little more deeper and find that the `n.Wrapped` here can also be other error types like  `errcode.Error` not only the `url.Error`. So I think we should make a judgement here to see if it was the `url.Error` and if so we do the unescape operation.